etna/transforms/__init__.py

```diff
@@ -1,41 +1,40 @@
+from etna.transforms.encoders import SegmentEncoderTransform
+from etna.transforms.feature_selection import FilterFeaturesTransform
+from etna.transforms.feature_selection import GaleShapleyFeatureSelectionTransform
+from etna.transforms.feature_selection import TreeFeatureSelectionTransform
+from etna.transforms.math import AddConstTransform
+from etna.transforms.math import BoxCoxTransform
+from etna.transforms.math import LagTransform
+from etna.transforms.math import LogTransform
+from etna.transforms.math import MaxAbsScalerTransform
+from etna.transforms.math import MaxTransform
+from etna.transforms.math import MeanTransform
+from etna.transforms.math import MedianTransform
+from etna.transforms.math import MinMaxScalerTransform
+from etna.transforms.math import MinTransform
+from etna.transforms.math import QuantileTransform
+from etna.transforms.math import RobustScalerTransform
+from etna.transforms.math import StandardScalerTransform
+from etna.transforms.math import StdTransform
+from etna.transforms.math import YeoJohnsonTransform
+from etna.transforms.missing_values import ResampleWithDistributionTransform
+from etna.transforms.missing_values import TimeSeriesImputerTransform
+from etna.transforms.nn import PytorchForecastingTransform
 from etna.transforms.outliers import DensityOutliersTransform
 from etna.transforms.outliers import MedianOutliersTransform
 from etna.transforms.outliers import PredictionIntervalOutliersTransform
 from etna.transforms.outliers import SAXOutliersTransform
-from etna.transforms.power import BoxCoxTransform
-from etna.transforms.power import YeoJohnsonTransform
-from etna.transforms.pytorch_forecasting import PytorchForecastingTransform
-from etna.transforms.resample import ResampleWithDistributionTransform
-from etna.transforms.scalers import MaxAbsScalerTransform
-from etna.transforms.scalers import MinMaxScalerTransform
-from etna.transforms.scalers import RobustScalerTransform
-from etna.transforms.scalers import StandardScalerTransform
-from etna.transforms.segment_encoder import MeanSegmentEncoderTransform
-from etna.transforms.segment_encoder import SegmentEncoderTransform
-from etna.transforms.special_days import SpecialDaysTransform
-from etna.transforms.statistics import MaxTransform
-from etna.transforms.statistics import MeanTransform
-from etna.transforms.statistics import MedianTransform
-from etna.transforms.statistics import MinTransform
-from etna.transforms.statistics import QuantileTransform
-from etna.transforms.statistics import StdTransform
-from etna.transforms.stl import STLTransform
-from etna.transforms.trend import TrendTransform
+from etna.transforms.timestamp import DateFlagsTransform
+from etna.transforms.timestamp import FourierTransform
+from etna.transforms.timestamp import HolidayTransform
+from etna.transforms.timestamp import SpecialDaysTransform
+from etna.transforms.timestamp import TimeFlagsTransform
```

Comment:
There are 40 lines in a new file and 41 in the old version. It is strange.

Reply:
it's true
I didn't add import for `BaseFeatureSelectionTransform` at the top level of transforms because this class seems to be not very useful for users